addon/adapters/application.js

```diff
@@ -0,0 +1,181 @@
+import Ember from 'ember';
+import DS from 'ember-data';
+
+export default DS.RESTAdapter.extend({
+
+    defaultSerializer: '-parse',
```

Comment:
I understand what you are saying but these are new files so there is no history to maintain.  In my opinion there should be consistency between the formatting in the files so whatever format is decided on should be applied to all the files universally.